desc.en.hocon

```diff
@@ -7439,7 +7852,7 @@
       "desc": "Flow control."
     },
     "max_payload_size": {
-      "desc": "Maximum retained message size."
+      "desc": "The maximum size of retained messages allowed to be stored. EMQX will refuse to store retained messages larger than this size and output an Error log with the keyword 'retain_failed_for_payload_size_exceeded_limit'.\n\n0 means unlimited retained message size."
```

Comment:
The updates for max_payload_size, msg_clear_interval, msg_expiry_interval are not found in desc.zh.hocon